lsy_drone_racing/envs/race_core.py

```diff
@@ -354,9 +348,27 @@ def _reset(
         # explicitly do it here
         self.sim.reset(mask=mask)
         key, subkey = jax.random.split(self.sim.data.core.rng_key)
+        # Generate random track
+        self.track_gen_key, track_sub_key = jax.random.split(self.track_gen_key)
```

Comment:
You can split multiple keys from jax.random.split. It's more efficient to just generate three instead of two one call further up